lib/presentation/engage/bible_series/helper.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:wpa_app/domain/bible_series/entities.dart';
+import 'package:wpa_app/domain/completions/entities.dart';
+
+bool isOnTime(Timestamp date) {
```

Comment:
Can we write a unit test for this function? I think `DateTime.now()` would have a timestamp which could throw off the response. Consider using the intl library which I am already using in `lib\presentation\common\date_formatter.dart`.